crates/cranelift/src/func_environ.rs

```diff
@@ -546,7 +573,11 @@ impl<'module_environment> FuncEnvironment<'module_environment> {
         // Compare to see if our fuel is positive, and if so we ran out of gas.
         // Otherwise we can continue on like usual.
         let zero = builder.ins().iconst(ir::types::I64, 0);
-        let fuel = builder.use_var(self.fuel_var);
+        let fuel = if self.tunables.outband_fuel {
```

Comment:
Similar to above I think that `fuel_check` is never executed with `outband_fuel`?